config/default/core.extension.yml

```diff
@@ -6,9 +6,10 @@ module:
   ckeditor: 0
   color: 0
   config: 0
+  config_filter: 0
+  config_split: 0
   contextual: 0
   datetime: 0
-  dblog: 0
```

Comment:
I think we should also enable `syslog` now that we have disabled this

Reply:
Added syslog to default config